pkg/cmd/synonyms/save/save.go

```diff
@@ -0,0 +1,123 @@
+package save
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/opt"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/search"
+	"github.com/spf13/cobra"
+
+	"github.com/algolia/cli/pkg/cmdutil"
+	"github.com/algolia/cli/pkg/config"
+	"github.com/algolia/cli/pkg/iostreams"
+	"github.com/algolia/cli/pkg/utils"
+)
+
+type SaveOptions struct {
+	Config config.IConfig
+	IO     *iostreams.IOStreams
+
+	SearchClient func() (*search.Client, error)
+
+	Indice            string
+	SynonymID         string
+	ForwardToReplicas bool
+	SynonymValues     []string
```

Comment:
After re-reading our docs for the API Clients, I wonder if `values` would be the right name for the flags / field here: https://www.algolia.com/doc/api-reference/api-methods/save-synonym/#examples

Could be just `Synonyms` for the field and `-s` / `--synonyms` for the flag. Would be closer to the API Clients methods. WDYT?

Reply:
Make sense, I agree!
I'll try to keep that in mind for the future 🙏